config/action.d/pf.conf

```diff
@@ -91,5 +90,5 @@ allports = any
 
 # Option: multiport
 # Notes.: addition to block access only to specific ports
-# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>]"
+# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>, port="%(port)s"]"
```

Comment:
You are wrong here: the `port` will be specified through `action_` or `action`, but no longer with `banaction`.